.circleci/config.yml

```diff
@@ -32,6 +32,16 @@ jobs:
             - node_modules
           key: v1-dependencies-{{ checksum "package.json" }}
 
+      - run:
+          name: "Wait for redis to become available"
+          command: |
+            docker start
```

Comment:
Not sure what the specific commands to use here in order to start the docker-compose, wait for it to start, then run tests

Reply:
Instead of using docker-compose, we can tell Circle to start multiple images, by adding new ones to the `docker` section of this file. So if we add the Redis image, it should become accessible from the tests (as long as the tests are using the default address/port).

Reply:
For an example, see the `test` job config on the public api repo.

Reply:
Relevant documentation: https://circleci.com/docs/2.0/databases/#using-dockerize-to-wait-for-dependencies

Reply:
Yay I added the dependency for the redis image and dockerize to wait for it. It seems like it works on the latest node, but it fails to start on Node 8 & 10, not sure if it's a compatibility issue 🤔